includes/admin/class-wc-admin-meta-boxes.php

```diff
@@ -185,15 +185,20 @@ public function rename_meta_boxes() {
 	/**
 	 * Check if we're saving, the trigger an action based on the post type.
 	 *
-	 * @param  int $post_id
+	 * @param  int    $post_id
 	 * @param  object $post
+	 * @param  bool   $update
 	 */
-	public function save_meta_boxes( $post_id, $post ) {
+	public function save_meta_boxes( $post_id, $post, $update ) {
 		// $post_id and $post are required
 		if ( empty( $post_id ) || empty( $post ) || self::$saved_meta_boxes ) {
 			return;
 		}
 
+		if ( ! $update ) {
+			update_post_meta( $post_id, '_woocommerce_force_meta_update', true );
```

Comment:
This is where it's created/updated to do its job. 
Don't we need to delete it later?